tests/test_simulator.py

```diff
@@ -282,3 +282,13 @@ def test_evovle_phase():
     simu.set_circuit(c)
     output_sv = simu.evolve(input_state)
     assert output_sv[BasicState([1, 1])] == pytest.approx(complex(math.sqrt(2)/4, math.sqrt(6)/4))
+
+
+def test_simulator_evolve_svd():
```

Comment:
Do you think you can add a more complex test for `evolve_svd` with post-selection?